pulp_rpm/app/tasks/publishing.py

```diff
@@ -103,10 +103,21 @@ def publish_artifacts(self, content):
         """
         published_artifacts = []
         for content_artifact in ContentArtifact.objects.filter(
-                content__in=content.exclude(pulp_type="rpm.repo_metadata_file").distinct()
+                content__in=content.exclude(
+                    pulp_type="rpm.repo_metadata_file"
```

Comment:
the type can be taken from the classmethod, not to hard code it.
 RepoMetadataFile.get_pulp_type()